create-project

```diff
@@ -1,79 +1,70 @@
 #! /usr/bin/env python3
 
 import re
-import sys
-from shutil import copyfile
-
-
-def create_service(domain):
-    service_name = domain.replace('.', '-')
-    copyfile('templates/service.yaml', f'services/{service_name}.yaml')
-    sentry_dsn = input("What's the SENTRY_DSN string? ").strip()
-    update_file(
-        f'services/{service_name}.yaml',
-        {
-            'service_name': service_name,
-            'domain': domain,
-            'sentry_dsn': sentry_dsn
-        }
-    )
+import yaml
```

Comment:
This project is now going to need a `requirements.txt` with at least `pyyaml` in it.